syntax = "proto3";

package calibration;

message PredictionList {
  repeated float values = 1;
}

message Predictions {
  repeated PredictionList pred = 1;
  repeated int32 labels = 2;
  bool eof = 3;
  int32 batch_index = 4;
}



